addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/InformationDisclosureSuspiciousCommentsScanRule.java

```diff
@@ -107,19 +110,21 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
                                             MESSAGE_PREFIX + "otherinfo", pattern, elStr));
                             todoComments.append("\n");
                             confidence = Alert.CONFIDENCE_LOW;
+                            evidence.add(elStr);
                             break; // Only need to record this script once
                         }
                     }
                     offset = el.getEnd();
                 }
             }
-            if (todoComments.length() > 0) {
-                this.raiseAlert(msg, id, todoComments.toString(), confidence);
+            for (String e : evidence) {
```

Comment:
This is now raising an alert per comment, was that on purpose? The other info field will contain the information of all comments not just the one that matches the evidence.

Reply:
Yep, I thought that makes more sense to have one alert per comment, do you agree? 
The other contains the matching pattern, which is not that helpful...

Reply:
IMO it's easier to manage with a single alert per message.

Reply:
I was just thinking about false positives - you might want one alert per comment so you can ignore only the alerts that are false positive

Reply:
Worth mentioning this in the change log as its a change in the way the rule works.

Reply:
Yep, I updated the change log

Reply:
Did you push the changes? I'm still not seeing it :/